Sources/BlogModule/BlogRouter.swift

```diff
@@ -25,6 +25,15 @@ struct BlogRouter: FeatherRouter {
         adminRoutes.register(authorLinkController)
     }
     
+    func apiRoutesHook(args: HookArguments) {
+        let apiRoutes = args.routes
+
+        apiRoutes.registerPublicApi(postController)
```

Comment:
Well, this is not a good approach for models with metadata support. 
If we use the default public route implementation we'll expose unpublished / archived items in the public API.
I'd like to avoid that, so we have to override the endpoints & query using the appropriate metadata (status & date).

Reply:
The output of the public API endpoints should be the same as the web frontend has (except it's not HTML, but JSON of course). Ideally it should work the same as it is implemented in the frontend controller.

Apart from this, thank you very much for these contributions. 👍

Reply:
@tib that's correct. Changed a bit the Core and now it should. 
https://github.com/FeatherCMS/feather-core/pull/9